src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/HandlerProvider.cs

```diff
@@ -50,14 +50,14 @@ public ImmutableArray<RequestHandlerMetadata> GetRegisteredMethods()
         return requestHandlers.Keys.ToImmutableArray();
     }
 
-    private ImmutableDictionary<RequestHandlerMetadata, Lazy<IMethodHandler>> GetRequestHandlers()
+    private ImmutableDictionary<RequestHandlerMetadata, List<Lazy<IMethodHandler, string?>>> GetRequestHandlers()
```

Comment:
can you remind me - the string is essentially the language name right?

Reply:
Correct, it's the language name provided by the LSP client